dynamiqs/integrators/core/interfaces.py

```diff
@@ -16,14 +20,25 @@ class SEInterface(eqx.Module):
     H: TimeArray
 
 
-class MEInterface(eqx.Module):
+class _MInterface(eqx.Module):
+    def L(self, t: Scalar) -> list[QArray]:
+        return [_L(t) for _L in self.Ls]  # (nLs, n, n)
```

Comment:
This calls `self.Ls` which is not defined in this context (only defined in child classes).

I think I would keep `MEInterface` as before, and inherit `MCInterface` from it. The MC solver is still solving the ME, so it's not unexpected IMO.